### PR TITLE
ADR messaging bus

### PR DESCRIPTION
**Pull request by Tezno**

### CODE REVIEW

File: docs/ADR/009-messaging-bus.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# ADR-008-open-source-license
```
Comment (RaboliotLeGris) [inline docs/ADR/009-messaging-bus.md:1]: It will be better if this file is in another PR

Comment (Tezno) [reply docs/ADR/009-messaging-bus.md:1]: ooh bad handling

File: docs/ADR/009-messaging-bus.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# ADR-009-messageging-bus
+
+- Creation Date: 11/02/2022
+- Status: DRAFT
+
+## Context
+
+The Voogle project is a demonstrator for Squarscale. The software is not intended to be commercialized. We decided that the software would be open source. We must choose the best license for our use
```
Comment (RaboliotLeGris) [inline docs/ADR/009-messaging-bus.md:8]: You forgot to change that :)

File: docs/ADR/009-messaging-bus.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+### 2. Redis
+
+Redis (Remote Dictionary Server) is an in-memory data structure store, used as a distributed, in-memory key–value database, cache and message broker, with optional durability. Redis supports different kinds of abstract data structures, such as strings, lists, maps, sets, sorted sets, HyperLogLogs, bitmaps, streams, and spatial indices. The project was developed and maintained by Salvatore Sanfilippo. From 2015 until 2020, he led a project core team sponsored by Redis Labs. Salvatore Sanfilippo left Redis as the maintainer in 2020. It is open-source software released under a BSD 3-clause license. In 2021, not long after the original author and main maintainer left, Redis Labs dropped the Labs from its name and now redis, the open source DB as well as Redis Labs, the commercial company, are referred to as "redis".
+[Wikipedia](https://en.wikipedia.org/wiki/Redis)
+
+#### Benefits
+
+- Already available on Sqsc
+- Available on AWS with Amazon ElastiCache for Redis
+- DB most popular
```
Comment (RaboliotLeGris) [inline docs/ADR/009-messaging-bus.md:47]:
```suggestion
-  Most popular DB
```

File: docs/ADR/009-messaging-bus.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# ADR-009-messageging-bus
+
+- Creation Date: 11/02/2022
+- Status: DRAFT
+
+## Context
+
+The Voogle project is a demonstrator for Squarscale. The software is not intended to be commercialized. We decided that the software would be open source. We must choose the best license for our use
+
+## Decision
```
Comment (RaboliotLeGris) [inline docs/ADR/009-messaging-bus.md:10]: What is our decision ? RabbitMQ ?

File: docs/ADR/009-messaging-bus.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+# ADR-009-messageging-bus
+
+- Creation Date: 11/02/2022
+- Status: DRAFT
```
Comment (RaboliotLeGris) [inline docs/ADR/009-messaging-bus.md:4]:
```suggestion
- Status: Accepted
```

**Review by RaboliotLeGris — APPROVED** [review]

**Pull request merged**